lib.typ

```diff
@@ -12,17 +12,21 @@
 /// Output selected Chinese lorem ipsum paragraphs.
 ///
 /// *Example 1:*
-/// Cut some paragraphs from Lu Xun's _Zhufu_（《祝福》）.
-/// #example(`#kouhu(builtin-text: "zhufu", offset: 5, indicies: (2, 18), length: 31, between-para: "——")`, mode: "markup")
+/// Cut the first several characters of a paragraph from Lu Xun's _Zhufu_（《祝福》）.
+/// #example(`#kouhu(builtin-text: "zhufu", index: 18, length: 99)`, mode: "markup")
 ///
 /// *Example 2:*
+/// Cut multiple paragraphs from Lu Xun's _Zhufu_（《祝福》）.
+/// #example(`#kouhu(builtin-text: "zhufu", offset: 5, index: (2, 18), length: 31, between-para: "——")`, mode: "markup")
+///
+/// *Example 3:*
 /// Repeat some text until the specified length.
 /// #example(`#kouhu(custom-text: ("奥利",), between-para: none, length: 31)`, mode: "markup")
 /// - builtin-text (string): Name of the builtin text, see @@builtin-text-list() for a full list and length.
 /// - custom-text (array): Custom text to use. If not `none`, `builtin-text` will be ignored.
 /// - offset (int): Offset of the paragraph to start from.
-/// - indicies (array): Indicies (*NOT RANGE*) of paragraphs to use (`offset` will be added). `none` means all paragraphs. Any out-of-bound index will be ignored.
-/// - length (int): Length of graphme (characters) to print. `kouhu` will repeat over selected paragraphs until `length` is reached, and the final paragraph will likely be truncated. 0 for unlimited, i.e. print all selected paragraphs for only once.
+/// - index (int, array): Indices (*NOT RANGE*) of paragraphs to use (`offset` will be added). A single integer means a single paragraph, an array means multiple paragraphs, and `none` means all paragraphs. Any out-of-bound index will be ignored.
```

Comment:
`int / array` would be better, not to be confused as a tuple.

Reply:
Unfortunately, https://typst.app/universe/package/tidy/0.2.0/ defines the syntax to be `(int, array)`.

Their doc:
```typst
/// - param2 (content, length): This is param2.
#let something(param1, param2: 3pt, ..options) = { return true }
```

If I change it to `int / array`, then tidy v0.2.0 refuses to parse it:
![](https://github.com/user-attachments/assets/ce23b9da-5f98-4780-81ac-068326409571)
(compare it with the screenshot I pasted in this PR's main body)

https://typst.app/universe/package/tidy/0.4.0/ changes the syntax to `int | array`, which is clearer. However, tidy v0.4.0 conflicts with mantys v0.1.4…

```typst
#let sinc(
  /// The argument for the cardinal sine function. 
  /// -> int | float
  x
) = if x == 0 {1} else {calc.sin(x) / x}
```

Reply:
I have created https://github.com/jneug/typst-mantys/issues/24, and will try to fix it in the upstream. It will take some days/weeks/months, however.

Reply:
> tidy conflicts with mantys v0.1.4…

This is why I am stuck on tidy 0.2.x 😞